pkg/sql/sem/builtins/builtins.go

```diff
@@ -4558,6 +4558,28 @@ value if you rely on the HLC for accuracy.`,
 		},
 	),
 
+	"crdb_internal.active_version": makeBuiltin(
+		tree.FunctionProperties{Category: categorySystemInfo},
+		tree.Overload{
+			Types:      tree.ArgTypes{},
+			ReturnType: tree.FixedReturnType(types.Jsonb),
+			Fn: func(ctx *tree.EvalContext, _ tree.Datums) (tree.Datum, error) {
+				activeVersion := ctx.Settings.Version.ActiveVersionOrEmpty(ctx.Context)
+				jsonStr, err := gojson.Marshal(&activeVersion.Version)
```

Comment:
does this behave correctly if "empty" was returned by `ActiveVersionOrEmpty`? (asking mostly to be educated, since i don't quite know when the version would be empty.)

Reply:
it's empty if for some reason this function is called before the cluster is done initializing. In the SQL layer, that's about impossible since we don't let SQL start before cluster init.